### ISSUE TITLE
SqlServer provider - MathRoundTranslator

### ISSUE THREAD

**Issue by bubibubi** [aspnet/EntityFramework#2728]
In EF7 SQL Server provider, MathRoundTranslator (but also other Math related translators) the source code is similar to this (I added only some comments):

```C#
public virtual Expression Translate([NotNull] MethodCallExpression methodCallExpression)
{
    // Retrieve all methods with 1 parameter or with 2 parameters where the first parameter is an int
    var methodInfos = typeof(Math).GetTypeInfo().GetDeclaredMethods("Round").Where(m =>
            m.GetParameters().Count() == 1
            || (m.GetParameters().Count() == 2 && m.GetParameters()[1].ParameterType == typeof(int)));

    // Is the methodCallExpression one of the method retrieved
    if (methodInfos.Contains(methodCallExpression.Method))
    {
        // TODO: Check with a test case. The problem here could be that the second parameter is always a 0 also if we specify 2 parameters
        var arguments = new[] { methodCallExpression.Arguments[0], Expression.Constant(0) };

        return new SqlFunctionExpression("Round", arguments, methodCallExpression.Type);
    }

    return null;
}
```

In both overloads arguments is always 
methodCallExpression.Arguments[0], Expression.Constant(0)

In case of one parameter I can understand it (ROUND in T-SQL expects 2 parameters) but in case of two parameters I expected 
methodCallExpression.Arguments[0], methodCallExpression.Arguments[1]

How can it work so implemented?

**Comment by divega** [comment]
@maumar can you take a look?

**Comment by maumar** [comment]
@bubibubi  is correct, this is a bug

**Comment by anpete** [comment]
Related to #2730

**Comment by maumar** [comment]
Fixed in 71fe9fd2fa1081f8b66ef219a1eef93830b9cd54
